src/promptflow/promptflow/_telemetry/telemetry.py

```diff
@@ -0,0 +1,56 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import logging
+import os
+
+from promptflow._sdk._configuration import Configuration
+from promptflow._telemetry.logging_handler import get_appinsights_log_handler
+
+TELEMETRY_ENABLED = "TELEMETRY_ENABLED"
+PROMPTFLOW_LOGGER_NAMESPACE = "promptflow._telemetry"
+
+
+class TelemetryMixin(object):
+    def __init__(self, **kwargs):
+        # Need to call init for potential parent, otherwise it won't be initialized.
+        super().__init__(**kwargs)
+
+    def _get_telemetry_values(self, *args, **kwargs):  # pylint: disable=unused-argument
+        """Return the telemetry values of object.
+
+        :return: The telemetry values
+        :rtype: Dict
+        """
+        return {}
+
+
+def is_telemetry_enabled():
+    """Check if telemetry is enabled. User can enable telemetry by
+    1. setting environment variable TELEMETRY_ENABLED to true.
+    2. running `pf config set cli.telemetry_enable=true` command.
+    If None of the above is set, will prompt an input to ask user to enable telemetry.
```

Comment:
where do we prompt for this?

Reply:
Good catch, we won't do this accoding to last sync, let me change the doc string.